README.md

```diff
@@ -1,4 +1,5 @@
 # lintr
+[![R build status](https://github.com/jimhester/lintr/workflows/R-CMD-check/badge.svg)](https://github.com/jimhester/lintr/actions)
 [![Travis-CI Build Status](https://travis-ci.org/jimhester/lintr.svg?branch=master)](https://travis-ci.org/jimhester/lintr)
```

Comment:
Remember to remove the Travis badge when you take away `.travis.yml`

Reply:
done